hokusai/production.yml

```diff
@@ -164,7 +164,7 @@ spec:
 apiVersion: batch/v2alpha1
 kind: CronJob
 metadata:
-  name: horizon-refresh-cron
+  name: horizon-refresh-comparisons-cron
```

Comment:
Note that I think renaming the cron will create a _new_ cron upon the next deploy, and the old one will need to be manually deleted. Ditto in staging.

Reply:
Thanks for heads up -- wanted to make these names more specific now that we have multiple, will add #migration steps.